flocker/control/test/configurations/configuration_v4.py

```diff
@@ -0,0 +1,11 @@
+# Copyright ClusterHQ Inc.  See LICENSE file for details.
+
+# Generate a v4 configuration.
+# Hash to recreate: e99b89c0a7c036e9d50f8705871ded0c829d1a83
```

Comment:
:grey_question: Reminder to update this and the above file if you change the name of persistent state.

Almost seems like we should check these tests in after we have merged a commit that adds the new version. That way if we iterate on what "v4" looks like we don't run the risk of forgetting to update the json file.